clang/tools/libclang/CDependencies.cpp

```diff
@@ -640,3 +640,176 @@ std::string OutputLookup::lookupModuleOutput(const ModuleDeps &MD,
     PCMPath.first->second = ::lookupModuleOutput(MD, MOK, MLOContext, MLO);
   return PCMPath.first->second;
 }
+
+namespace {
+struct DependencyScannerReproducerOptions {
```

Comment:
We need to copy the strings/arrays so that their lifetime is maintained by the options object.